README.md

```diff
@@ -31,6 +31,9 @@ DATABASE_URL=postgresql://john.smith@localhost/laa-claim-non-standard-magistrate
 
 After you've defined your DB configuration in the above files, run the following:
 
+> NOTE: you need to manually instal postgis for the migrations to run. This can be done with
```

Comment:
I don't think this is true - postgis is included in the brewfile now

Reply:
might be my version of postgres... I will check I'm on the latest